apps/frontend/src/assets/css/scaffolding.css

```diff
@@ -31,3 +34,38 @@ body {
 	white-space: nowrap;
 	border: 0;
 }
+
+/* circles */
+.circle {
+	width: 30px;
+	height: 30px;
+	border-radius: 100%;
+}
+
+.circle-blue {
+	background: var(--primary-gradient);
+}
+
+.circle-pink {
+	background: var(--pink-gradient);
+}
+
+/* buttons */
+.btn {
+	width: 100%;
+	padding: 10px 0;
+	font-family: var(--nunito-font);
+	font-size: 16px;
+	font-weight: 700;
+	color: white;
```

Comment:
You have a css variable called --dark, which is essentially black color. If I were you I would double down on this practice and defined another css variable for color white

Reply:
done